### DISCUSSION TITLE
UI Readme Dark Mode

### DISCUSSION THREAD

**Discussion by onosendi** [verdaccio/verdaccio#3942] [Q&A] [answered]
I have the UI set to light mode, but the readme is still dark mode.

<img width="1210" alt="Screenshot 2023-07-19 at 09 50 03" src="https://github.com/verdaccio/verdaccio/assets/2631623/5ac2f271-ff0c-436d-96f8-a87839d3ec47">

**Comment by juanpicado** [comment]
It is a known issue no solution yet , the issue is with the package `github-markdown-css`, the UI uses the light theme by default, so the switch to dark mode is not done automatically, we need some logic to switch CSS from that library, more info here https://github.com/sindresorhus/github-markdown-css/issues/104 for possible solutions

**Comment by juanpicado** [comment, marked as answer]
https://github.com/verdaccio/verdaccio/pull/4071
